files/KoreBuild/modules/projectbuild/module.targets

```diff
@@ -75,6 +75,19 @@ Executes /t:{Target} on all projects
           ItemName="_ProjectToRestoreWithNuGet" />
     </MSBuild>
 
+    <MSBuild Targets="_RequiresCustomRestore"
```

Comment:
For naming consistency with how NuGet implemented this, could we name this target `_IsCustomRestoreTargetSupported`?